resources/views/index.blade.php

```diff
@@ -6,11 +6,17 @@
 
         window.laravelCookieConsent = (function () {
 
+            var COOKIE_VALUE = 1;
+
             function consentWithCookies() {
-                setCookie('{{ $cookieConsentConfig['cookie_name'] }}', 1, 365 * 20);
+                setCookie('{{ $cookieConsentConfig['cookie_name'] }}', COOKIE_VALUE, 365 * 20);
                 hideCookieDialog();
             }
 
+            function cookieExists(name) {
+                return (document.cookie.split(';').indexOf(name + '=' + COOKIE_VALUE) == 0);
```

Comment:
Correct me if I'm wrong, but I think this will only work if the cookie is the first retrieved cookie.